bob-backend/src/pearl/disk_controller.rs

```diff
@@ -538,14 +538,14 @@ impl DiskController {
         cnt
     }
 
-    pub(crate) async fn index_memory(&self) -> usize {
+    pub(crate) async fn active_index_memory(&self) -> usize {
```

Comment:
Looks like this function should return full memory occupied by indexes, because it is used only in metrics